doc-content/enterprise-only/optimizer/optimizer-running-webexamples-proc.adoc

```diff
@@ -0,0 +1,25 @@
+[id='optimizer-running-webexamples-proc']
+= Running the web examples
+Besides the GUI examples, {PRODUCT} also includes a set of web examples for {PLANNER}. The web examples include:
+
+* Vehicle routing: calculating the shortest possible route to pick up all items required for a number of different customers using either http://leafletjs.com/[Leaflet] or http://google.com/maps[Google Maps] visualizations. 
+* Cloud balancing: Assigning processes across computers with different specifications and costs. 
+
+.Prerequisites
+
+You have downloaded and unpacked the {PRODUCT} add-ons package, which includes the examples. For instructions about these actions, see <<examples-download-proc>>.
+
+The web examples require several JEE APIs to run, such as:
```

Comment:
to run:

Reply:
Removing "such as" would mean the list is complete. Not doing without SME input.

Reply:
Good point.